WooCommerce/Classes/ViewRelated/Dashboard/Settings/CardReaders/CardReaderSettingsConnectViewController.swift

```diff
@@ -0,0 +1,8 @@
+import UIKit
+
+class CardReaderSettingsConnectViewController: UIViewController {
```

Comment:
I understand this is still a placeholder. 

Unless you really want to subclass this (and in that case I'd argue there might be other solutions worth exploring) I would suggest defaulting to marking classes as final.

Back in the day [Apple suggested declaring classes as final also had performance implications](https://developer.apple.com/swift/blog/?id=27). I am not sure if that still holds true, though.

Reply:
Fixed in https://github.com/woocommerce/woocommerce-ios/pull/3699/commits/46c1941a4b75b3595b156f57b130316575113c0e